.github/workflows/release-please.yml

```diff
@@ -14,29 +15,49 @@ jobs:
       pull-requests: write
 
     steps:
-      - uses: google-github-actions/release-please-action@v3
+      - uses: google-github-actions/release-please-action@v4
         id: release
         with:
-          command: manifest
           token: ${{ secrets.GITHUB_TOKEN }}
-          default-branch: main
 
       - uses: actions/checkout@v4
         if: ${{ steps.release.outputs.releases_created }}
+
+      - name: Determine semantic version and PR branch from release-please output
+        if: ${{ steps.release.outputs.prs_created == 'true' }}
+        id: release-metadata
+        shell: bash
+        run: |
+          echo "semver=${{ steps.release.outputs.major }}.${{ steps.release.outputs.minor }}.${{ steps.release.outputs.patch }}" >> $GITHUB_OUTPUT
+          echo "branch=${{ fromJSON(steps.release.outputs.pr).headBranchName }}" >> $GITHUB_OUTPUT
+
+      - name: Update launchdarkly-server-sdk rockspec
+        uses: ./.github/actions/update-versions
+        if: ${{ steps.release-metadata.outcome == 'success' }}
+        with:
+          package: launchdarkly-server-sdk
+          branch: ${{ steps.release-metadata.outputs.branch }}
+          version: ${{ steps.release-metadata.outputs.semver }}
+
+      - name: Update launchdarkly-server-sdk-redis rockspec
+        uses: ./.github/actions/update-versions
+        if: ${{ steps.release-metadata.outcome == 'success' }}
         with:
-          fetch-depth: 0 # If you only need the current version keep this.
+          package: launchdarkly-server-sdk-redis
+          branch: ${{ steps.release-metadata.outputs.branch }}
+          version: ${{ steps.release-metadata.outputs.semver }}
 
       - name: Build and Test
-        if: ${{ steps.release.outputs.releases_created }}
+        if: ${{ steps.release.outputs.release_created }}
```

Comment:
So, does releases_created not work anymore? Because it is supposed to, so that may be another bug? Or does release_created not have the `== 'true'` but, but releases_creates does?

Reply:
Oh, the bug is `releases_created` specifically.

Reply:
🤦